docker/docker-compose.yml

```diff
@@ -19,6 +19,9 @@ services:
       - NODE_ENV
       - API_HOST
       - STATUS_PORT
+      - STATUS_URL
+      - POSTS_URL=http://posts
+      - POSTS_PORT=5555
```

Comment:
You shouldn't need the port.  That's only for internal use.  You should access Posts via our microservice router (Traefik).  Every other service that uses the Posts service just works with the public `POSTS_URL`, for example: https://github.com/Seneca-CDOT/telescope/blob/master/config/env.development#L131 and https://github.com/Seneca-CDOT/telescope/blob/master/config/env.production#L121.

Reply:
OK, please do it like this then https://github.com/Seneca-CDOT/telescope/blob/master/docker/development.yml#L107